examples/webgl_lights_spotlight.html

```diff
@@ -95,8 +99,8 @@
 				spotLight.map = textures[ 'disturb.jpg' ];
 
 				spotLight.castShadow = true;
-				spotLight.shadow.mapSize.width = 1024;
-				spotLight.shadow.mapSize.height = 1024;
+				spotLight.shadow.mapSize.width = 512;
+				spotLight.shadow.mapSize.height = 512;
```

Comment:
When comparing the new and previous version, I notice that the edges of the statue's shadows flickering now more noticeably. 

Did you lower the resolution to soften the shadows?

Reply:
Oh, I didn't realize we used such large values in the examples.

I'll be glad to restore it.

Reply:
done!